hpx/runtime/threads/cpu_mask.hpp

```diff
@@ -184,42 +184,21 @@ namespace hpx { namespace threads
 #define HPX_CPU_MASK_PREFIX "0x"
 #endif
 
-    inline bool equal(mask_cref_type lhs, mask_cref_type rhs, std::size_t numbits)
+    inline bool equal(mask_cref_type lhs, mask_cref_type rhs, std::size_t)
     {
-        for (std::size_t j = 0; j != numbits; ++j)
-        {
-            if (test(lhs, j) != test(rhs, j))
-            {
-                return false;
-            }
-        }
-        return true;
+        return lhs == rhs;
```

Comment:
It does: http://www.boost.org/doc/libs/1_63_0/libs/dynamic_bitset/dynamic_bitset.html#member-functions